homeassistant/bootstrap.py

```diff
@@ -403,6 +404,12 @@ def async_from_config_dict(config: Dict[str, Any],
         new_config[key] = value or {}
     config = new_config
 
+    # Merge packages into the main config
+    try:
+        config = merge_packages_config(config)
+    except vol.Invalid as ex:
```

Comment:
[This](https://github.com/home-assistant/home-assistant/pull/5140/files#diff-f45e3d5ea33826f22285843d05df829aR36) should be able to raise it